### PR TITLE
Fix method name

### PR DESCRIPTION
**Pull request by StijnSimons**

### CODE REVIEW

**Comment by StijnSimons** [comment]
#39

File: src/main/java/org/itxtech/nemisys/Client.java [hunk 1]
```diff
@@ -401,11 +401,16 @@ public void close(String reason, boolean needPk, byte type) {
 
         this.closeAllPlayers(reason, this.transferOnShutdown ? this.server.getFallbackClient() : null);
     }
-
-    public void sendPluginMesssage(String channel, byte[] data) {
+
```
Comment (wode490390) [inline src/main/java/org/itxtech/nemisys/Client.java]: space...

**Comment by StijnSimons** [comment]
I did this on the mobile editor... Show some respect

**Comment by SupremeMortal** [comment]
As in the issue, I think it would be best to keep the spelling mistake but deprecate otherwise it will break all plugins that use it.

**Comment by KCodeYT** [comment]
He has the deprecated method

**Pull request merged**